app.js

```diff
@@ -301,9 +301,11 @@ $(document).ready(function () {
 
     APP.translation.init(settings.getLanguage());
 
-    APP.API.init(APP.tokenData.externalAPISettings);
-    // For remote control testing:
-    // APP.API.init({forceEnable: true});
+    APP.API.init(APP.tokenData.jwt ? {
```

Comment:
I was making changes to API.init to be called multiple time which is needed by remote control and decided that this code doesn't belong to tokenData module. I'll add comments.